### DISCUSSION TITLE
Random Spinner

### DISCUSSION THREAD

**Discussion by bee-san** [Textualize/rich#1251] [Q&A]
Hey 👋🏻  Is it possible to randomise what spinner is used?

Like:

```py
with console.status("[bold green]Working...", spinner='random') as status:
```

I think it'd be rather neat & quirky 😄

**Comment by willmcgugan** [comment]
Don't think its worth adding to the core lib*, but you could grab the names from [_spinner.py](https://github.com/willmcgugan/rich/blob/master/rich/_spinners.py) if you wanted to implement it.

*might change my mind if this reply gets 10 👍
